### ISSUE TITLE
Name of sigma parameter change when going from proportional error to additive error

### ISSUE THREAD

**Issue by rikardn** [pharmpy/modelbuilder#125]
This is probably a bug in Pharmpy.

**Comment by antoniaih** [comment]
Yes, this happens in Pharmpy because there is already a parameter called 'sigma' when using `create_basic_pk_model`. Thus the new sigma is called 'sigma1'. This issue should probably be moved to Pharmpy.
